### PR TITLE
gh-114967: Fix "Built-in Exceptions" documentation ambiguous wording

### PR DESCRIPTION
**Pull request by HarryLHW**
In the documentation of "Built-in Exceptions", the second paragraph states `The built-in exceptions listed below can be generated by the interpreter or built-in functions`. Because of the new sections "Exception context" and "Inheriting from built-in exceptions" were introduced in Python 3.9, the word "below" became ambiguous.  Change 'below' to 'in this chapter'.

* Issue: gh-114967

----
📚 Documentation preview 📚: https://cpython-previews--114968.org.readthedocs.build/

### CODE REVIEW

**Comment by HarryLHW** [comment]
Sorry about the force-push. I mistakenly used the wrong email.

File: Doc/library/exceptions.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ equivalent, even if they have the same name.
 
 .. index:: pair: statement; raise
 
-The built-in exceptions listed below can be generated by the interpreter or
+The built-in exceptions listed in this document can be generated by the interpreter or
```
Comment (terryjreedy) [inline Doc/library/exceptions.rst]:
```suggestion
The built-in exceptions listed in this chapter can be generated by the interpreter or
```

**Comment by terryjreedy** [comment]
See https://github.com/python/cpython/issues/114967#issuecomment-1925441730.  Let's see if anyone else comments.

**Comment by HarryLHW** [comment]
> See [#114967 (comment)](https://github.com/python/cpython/issues/114967#issuecomment-1925441730). Let's see if anyone else comments.

Thank you :)

**Pull request merged**